whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/InitialSnapshotState.java

```diff
@@ -0,0 +1,8 @@
+package net.ripe.db.nrtm4.dao;
+
+import java.util.List;
+
+
+public record InitialSnapshotState(int serialId, List<RpslObjectData> rpslObjectData) {
+
```

Comment:
Is this record only for initial snapshot ? If not can we rename it to snapshotState?

Reply:
Can we also move it to domain package?